sw/device/tests/aon_timer_irq_test.c

```diff
@@ -21,19 +22,21 @@
 #include "sw/device/lib/testing/test_framework/check.h"
 #include "sw/device/lib/testing/test_framework/ottf_main.h"
 
-#include "hw/top_earlgrey/sw/autogen/top_earlgrey.h"
-
 OTTF_DEFINE_TEST_CONFIG();
 
-static const uint32_t kPlicTarget = kTopEarlgreyPlicTargetIbex0;
+static const uint32_t kPlicTarget = 0;
 static const uint32_t kTickFreqHz = 1000 * 1000;  // 1Mhz / 1us
 static dif_rv_core_ibex_t rv_core_ibex;
+static const dt_rv_core_ibex_t *rv_core_ibex_dt = &kDtRvCoreIbex[0];
```

Comment:
These are compile time constants, so should also be named using the pattern `kRvCoreIbexDt`.

Reply:
Good point, will address that in the next push